apps/consulting/components/BlogArticle/BlogPost/BlogPost.tsx

```diff
@@ -18,7 +18,7 @@ export const BlogPost: FC<TBlogPostProps> = ({ postText }) => {
 
   return (
     <div
-      className="prose-code:px-0 prose-pre:px-0 prose-img:mx-auto prose-figcaption:mt-[2rem] mb-[5rem] min-w-full text-[1.8rem] leading-[2.7rem] text-black prose-code:bg-transparent prose-pre:bg-transparent prose-code:rounded-lg prose sm:prose-figcaption:mt-0"
+      className="prose-code:px-0 prose-pre:px-0 prose-img:mx-auto prose-figcaption:mt-[2rem] mb-[5rem] min-w-full text-[1.8rem] leading-[2.7rem] text-black prose-code:bg-transparent prose-pre:bg-transparent prose-code:rounded-lg prose sm:prose-figcaption:mt-0 focus:prose-a:text-violet hover:prose-a:text-violet"
```

Comment:
In Consulting blog posts, change the link to purple when hovered or focussed.

Reply:
nit pick: can we add an offset to the underline on the links it makes it also easier to read 2-3px should do:

`text-underline-offset: 3px;` is what I have used in the past

Reply:
Definitely! I added the underline offset and it definitely looks nicer, thanks!